apps/meteor/ee/app/federation-v2/server/infrastructure/rocket-chat/hooks/index.ts

```diff
@@ -18,14 +18,15 @@ export class FederationHooksEE {
 		);
 	}
 
-	public static onUsersAddedToARoom(callback: (room: IRoom, inviter: IUser, addedUsers: IUser[]) => Promise<void>): void {
+	public static onUsersAddedToARoom(callback: (room: IRoom, inviter: IUser, addedUsers: IUser[] | string[]) => Promise<void>): void {
```

Comment:
```suggestion
	public static onUsersAddedToARoom(callback: (room: IRoom, inviter: IUser, addedUsers: IUser[] | Username[]) => Promise<void>): void {
```

Just to be explicit about what information is expected on this param.
Import Username from @rocket.chat/core-typings